docs/release-notes/highlights-0.9.0.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[release-highlights-0.9.0]]
+== 0.9.0 Release Highlights
+
+[float]
+=== Kubernetes Distribution Support
+
+In addition to Google Kubernetes Engine (GKE) and vanilla Kubernetes, ECK now also works on
+Azure Kubernetes Service (AKS), Amazon Elastic Kubernetes Service (EKS) and
+Red Hat OpenShift 3.11+ (with some restrictions on APM server functionality)
```

Comment:
I'm wondering if it's important to write APM server with a **s** in uppercase (applicable below).